docs/Release Notes.md

```diff
@@ -66,16 +66,21 @@ The Ansible Modules for Dell PowerScale support the following features:
 - Get details and modify SMB global settings.
 - Get details and modify SNMP settings.
 - Import, modify, setting default, and delete a server certificate.
+- Get details, create, modify, delete of auth roles.
```

Comment:
Get details, create, modify, and delete auth roles.

Reply:
yes, updated